expression/builtin_math.go

```diff
@@ -1975,6 +1978,10 @@ func (b *builtinTruncateIntSig) evalInt(row chunk.Row) (int64, bool, error) {
 	if d >= 0 {
 		return x, false, nil
 	}
+	// math.Pow10(d) == INF if d > 308
```

Comment:
Better add a case for d=308.

Reply:
new test was added, positive number do not run here. I add -308, -307 and -309